debian/changelog

```diff
@@ -1,3 +1,10 @@
+matrix-synapse-py3 (1.80.0+nmu1) UNRELEASED; urgency=medium
+
+  * Non-maintainer upload.
```

Comment:
this line shouldn't be here. it will not be a non-maintainer upload once it happens.